platforms/hyperledger-indy/charts/indy-auth-job/Chart.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-appVersion: 1
-description: A Helm chart for job indy-auth-job
+apiVersion: v2
```

Comment:
here only the appVersion change to "2.0" is required; same for all charts

Reply:
done